### PR TITLE
fix(ci): run release step for one matrix combo

### PR DESCRIPTION
**Pull request by PaulHax**

### CODE REVIEW

**Review by thewtex — APPROVED** [review]
`  `thanks!`  `

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -20,21 +20,21 @@ jobs:
...
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
+      - run: npm install
+      - run: npm run build --if-present
+      - run: npm run prettier
+      - name: Release
+        if: matrix.node-version == '18.x' && matrix.os == 'ubuntu-latest'
```
Comment (thewtex) [inline .github/workflows/publish.yml:33]: We should still have the check for the `main` branch here (do not try to run on pull requests).

**Review by thewtex — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (PaulHax) [reply .github/workflows/publish.yml:33]: Looks like semantic-release does the right thing without the branch check:
`[6:27:40 PM] [semantic-release] › ℹ  This run was triggered by a pull request and therefore a new version won't be published.`

Plus might work when tagging?

Comment (thewtex) [reply .github/workflows/publish.yml:33]: Ok, let's forward to semantic-release.

**Review by thewtex — APPROVED** [review]

**Pull request merged**